Whiskey/Functions/New-WhiskeySemanticVersion.ps1

```diff
@@ -22,6 +22,11 @@ function New-WhiskeySemanticVersion
         [object]
         $Version,
 
+        [AllowNull()]

```

Comment:
Users shouldn't be able to pass both a version and a path. They must be mutually-exclusive. Add parameter sets:

```
[Parameter(Mandatory=$true,ParameterSetName='ByVersion')]
[object]
$Version,

[Parameter(Mandatory=$true,ParameterSetName='ByPath')]
[string]
$Path
```

Reply:
Should `Version` allow null?

Reply:
Yes. It is how we know to use an auto-generated version.

Reply:
ok then this is Done